scripts/check_formatting

```diff
@@ -3,6 +3,10 @@
 set -o errexit
 set -o nounset
 
+
+# Apply buildifier to fix the BUILD and .bzl formatting.
+find oak examples toolchain -type f -exec buildifier -mode=fix {} +
```

Comment:
This script is supposed to just check formatting, but from this command line I'm guessing buildifier is actually formatting files if necessary, which is not what we want.

Reply:
Do you prefer check mode or print_if_changed mode? Check mode only shows files that need to be changd, while print_if_changed will show the diff.

Reply:
Nevermind, I see now that you have a separate file for actually applying the formatting.